selenium.js

```diff
@@ -99,9 +99,6 @@ const ignore = {
   }
 };
 
-// Sort array randomly
-const shuffle = () => Math.random() - 0.5;
```

Comment:
https://www.robweir.com/blog/2010/02/microsoft-random-browser-ballot.html might be amusing reading, this method of shuffling is imperfect and has been a high profile "browser choice" bug in the past!